projects/step-core/src/lib/components/toast/toast.component.html

```diff
@@ -0,0 +1,14 @@
+@if (message) {
+  <div class="toast">
+    <span class="success-icon">&#10003;</span>
```

Comment:
@neogucky @lukasz-lepa 
Shouldn't we use feather-icons here instead of unicode symbols?